client/my-sites/scan/style.scss

```diff
@@ -1,4 +1,4 @@
-.scan .formatted-header {
+.scan .formatted-header.is-left-align {
```

Comment:
I added specificity here to avoid a collision with rules for normal formatted headers, and to make it look the same as the Backup page.